stream_chat/tests/async_chat/test_channel.py

```diff
@@ -95,6 +95,17 @@ async def test_truncate(self, event_loop, channel):
         response = await channel.truncate()
         assert "channel" in response
 
+    @pytest.mark.asyncio
+    async def test_truncate_with_options(self, event_loop, channel, random_user):
```

Comment:
btw, I know we do this everywhere, but we don't need to include the `event_loop` fixture here if we don't use it in our tests.

Reply:
let's drop it separately globally

Reply:
yeah, I guess it's mandatory for some reason. I'll check if it works without it them remove from everywhere

Reply:
pytest injects them dynamically if you include them, similar to the custom fixtures `channel` and `random_user`. If you leave them out of the function signature, it should still work. (unless you use them in your test of course).

see: https://pythonrepo.com/repo/pytest-dev-pytest-asyncio-python-testing-codebases-and-generating-test-data

Reply:
https://github.com/GetStream/stream-chat-python/pull/87